.github/workflows/build.yml

```diff
@@ -181,8 +162,11 @@ jobs:
 
       - name: Dispatch build result
         uses: ./tmp/.github/actions/dispatch-build-result
+        env:
+          PackageNames: GeneXus.DeploymentTargets
```

Comment:
Can we calculate it from "Push packages" step?

Reply:
No without checking out the dispatch-build-result repo in the other job or this repo in this job.

The idea here was that since this repo is much simpler that the DotNet Classes one, there is no need to automate the calculation of packages to publish.

Reply:
Ok. Let's fix the build first. The thing was that we are already doing calculation on the generated packages in other step.